scripts/build-windows-k8s.sh

```diff
@@ -83,8 +83,8 @@ k8s_17_cherry_pick() {
 	# adeb88d774 Use adapter vEthernet (HNSTransparent) on Windows host network to find node IP
 	# 02549d6647 Merge pull request #50914 from shyamjvs/add-logging-to-logdump
 
-	git cherry-pick 02549d6647..45ba7bb0fb
-        git cherry-pick 51fab673e1..72b9c8f519
+	git cherry-pick 02549d6647^..45ba7bb0fb
```

Comment:
@martell Thanks for the suggestion. However, on A..B, we intentionally use a commit A before the commit we want to include. So for example, 02549d6647..45ba7bb0fb, we actually want to include the commit after 02549d6647, not 02549d6647. So if we go with your approach, we would need to change 02549d6647 to the commit after it.
@seanknox Please double check.

Reply:
hmm, I see. read my comment above regarding 79cf996.
I think A^..B and changing 02549d6647 to the commit after if that is what you want to do here because most developers won't be aware of this behaviour, increasing the barrier to contribution entry.

Reply:
I agree to use A^..B. Please change 79cf9963f7, etc. accordingly. We used to have 5fa0725025 (the one after 79cf9963f7) without ^ and  we fixed it. But your way is a better fix.